### PR TITLE
Add Polkadot Node for Westend Testnet

### PR DESCRIPTION
**Pull request by changsheng1239**
Dockerfile + DSL for deploying Polkadot Node for Westend Testnet on Akash

### CODE REVIEW

File: polkadot-testnet/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    polkadot-profile:
+      resources:
+        cpu:
+          units: 2.0
+        memory:
+          size: 8Gi
+        storage:
+          size: 64Gi
+  placement:
+    westcoast:
```
Comment (boz) [inline polkadot-testnet/deploy.yml:31]: Please add the `signedBy` section that you see in other examples.

**Review by boz — CHANGES_REQUESTED** [review]
Thanks so much @changsheng1239.

One change requested - this is good to go otherwise.

File: polkadot-testnet/deploy.yml (hunk 1 above)
Comment (changsheng1239) [reply polkadot-testnet/deploy.yml:31]: sorry about this late reply. the issue is resolved. `signedBy` added now

**Comment by changsheng1239** [comment]
sorry for the late reply. Added `signedBy` attribute, i think it's good to go now.

**Comment by andy108369** [comment]
@changsheng1239 please add two signedBy keys like here https://github.com/akash-network/awesome-akash/blob/b720cd57784f98eb471c5ec5f205ca68abe84fc2/Grav/deploy.yaml#L26-L29

And update the top-level README.md file.

Thank you.

**Pull request closed without merging**